drivers/bluetooth/hci/userchan.c

```diff
@@ -93,6 +95,67 @@ static struct net_buf *get_rx(const uint8_t *buf)
 	return NULL;
 }
 
+/**
+ * @brief Decode the length of an HCI H4 packet
+ * @details Decodes packet length according to Bluetooth spec v5.4 Vol 4 Part E
+ * @param packet_buff	Pointer to a HCI packet buffer
+ * @return Length of the HCI packet in bytes, zero if no valid packet found.
+ */
+static uint16_t packet_len(const uint8_t *packet_buff)
```

Comment:
Sorry if this is getting a bit overly pedantic, but I'd prefer if we stick to existing variable naming conventions and also avoid unnecessarily long names. This helps keep the code more readable (for someone who has read a lot of other Bluetooth subsystem code). and also avoids excessively long lines.

Just call the input parameter `const uint8_t *buf`

Reply:
Oh no problem, I am aware that style and naming conventions are highly dependent on organisations policies. As I am still not that familiar with the codebase of Zephyr I know this could come up.